_integration-test/ensure-backup-restore-works.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -ex
+
+source install/_lib.sh
+source install/dc-detect-version.sh
+
+echo "${_group}Test that backup/restore works..."
+echo "Creating backup..."
+backup_path="$(pwd)/sentry/backup"
+mkdir -p $backup_path
+# Docker was giving me permissioning issues when trying to create this file and write to it even after giving read + write access
+# to group and owner. Instead, try creating the empty file and then give everyone write access to the backup file
+touch $backup_path/backup.json
+chmod 666 $backup_path/backup.json
+$dcr -v $backup_path:/sentry-data/backup -T -e SENTRY_LOG_LEVEL=CRITICAL web export /sentry-data/backup/backup.json
```

Comment:
Would love to see this match [the docs](https://develop.sentry.dev/self-hosted/backup/#backup) exactly.

Reply:
With a comment saying as much.

Reply:
If you can't get it _exactly_ like the docs that's fine, just say so in the comment.